FRAuth/FRAuth/User/Browser.swift

```diff
@@ -158,7 +161,94 @@ import SafariServices
             completion(nil, NetworkError.invalidRequest("failed to generate /authorize request for external user agent"))
         }
     }
-    
+
+
+    //  MARK: - End Session
+
+    /// Performs external user-agent logout with /endSession request
+    /// - Parameter completion: Completion callback that returns an error on failure
+    @objc
+    public func logout(completion: @escaping UserCallback) {
+      browserMode = .logout
+
+      //perform browser logout only if signoutRredirectUri is provided
+      guard self.oAuth2Client.signoutRredirectUri != nil else {
+        completion(nil, nil)
+        self.cleanUp()
```

Comment:
I think that if the signoutRredirectUri == nil the logout will not do the AT revocation I think in any case we need to revoke AT

Reply:
Saying that if you call that Logout, do we revoke the refresh token or just clean up locally?

Reply:
the regular logout() is called in the completion block regardless of the outcome of this browser logout method. so AT is always revoked both when signoutRredirectUri is nil and when it isnt